convert2rhel/systeminfo.py

```diff
@@ -359,6 +364,80 @@ def corresponds_to_rhel_eus_release(self):
         """
         return self.releasever in EUS_MINOR_VERSIONS
 
+    def _get_dbus_status(self):
+        """
+        Check whether dbus is running.
+
+        :returns: True if dbus is running.  Otherwise False
+        """
+        retries = 0
+        status = False
+
+        while retries < CHECK_DBUS_STATUS_RETRIES:
+            if self.version.major <= 6:
+                status = _get_dbus_status_from_sysv()
+            else:
+                status = _get_dbus_status_from_systemd()
+
+            if status is not None:
+                # We know that DBus is definitely running or stopped
+                break
+
+            # Wait for 1 second, 2 seconds, and then 4 seconds for dbus to be running
+            # (In case it was started before convert2rhel but it is slow to start)
+            time.sleep(2**retries)
+            retries += 1
+
+        else:  # while-else
+            # If we haven't gotten a definite yes or no but we've exceeded or retries,
+            # report that DBus is not running
+            status = False
+
+        return status
+
+
+def _get_dbus_status_from_sysv():
+    """Get DBus status from SysVinit compatible systems."""
+    # None means the status should be retried because we weren't sure if it is turned off.
+    running = None
+    output, _ret_code = utils.run_subprocess(["/sbin/service", "messagebus", "status"])
+    for line in output.splitlines():
+        if line.startswith("messagebus"):
+            if "running" in line:
+                running = True
+                break
+
+            # Note: SysV has a stopped status but I don't think that toggles until after
+            # the service is running so we could be caught in the case where the service
+            # is starting if we don't retry.
+
+    return running
+
+
+def _get_dbus_status_from_systemd():
+    """Get DBus status from systemd."""
+    # Reloading, activating, etc will return None which means to retry
```

Comment:
We're not retrieving the status, but rather checking whether it's running
```suggestion
def _check_dbus_service_running():
    """
    Check that DBus systemd service is running.
    :returns: true/false if status is determined. None if status is indeterminate
    """
```

Reply:
The three functions involved in checking whether dbus is running have been renamed here: https://github.com/oamg/convert2rhel/pull/540/commits/f0f7ffd0f5b51b824abc6e0f80f3fa5a46c8352c